items/README.md

```diff
@@ -0,0 +1,15 @@
+# items firestore collection data model
+
+```
+{
+  'id': string,
+  'itemId': string,
+  'itemType': string,
+  'levelId': string,
+  'levelName': string,
+  'levelNumber': number,
```

Comment:
Can this be :

```
level: {
  id:
  name:
  value:
 }
 ```

Reply:
Created the refactor ticket here https://github.com/Real-Dev-Squad/website-backend/issues/974